chart/templates/configmap.yaml

```diff
@@ -14,3 +14,4 @@ data:
   init_container_configuration_endpoint: {{ include "job-executor-service.remote-control-plane.configuration-endpoint" . }}
   default_job_service_account: "{{ include "job-executor-service.jobConfig.serviceAccountName" . }}"
   allow_privileged_jobs: "{{ .Values.jobConfig.allowPrivilegedJobs | default "false" }}"
+  additional_job_labels: {{ .Values.jobConfig.labels | toJson | quote }}
```

Comment:
why do we convert the labels (which I am assuming is a map[string]string) to a json string ?

Reply:
I'm not sure if a yaml map can be used without any problems as environment variable, but I will try to avoid converting